validation-test/stdlib/String.swift

```diff
@@ -999,6 +1000,30 @@ StringTests.test(
 #endif
 }
 
+StringTests.test("String._fastCStringContents()")
+  .skip(.nativeRuntime("_NSContiguousString undefined without _runtime(_ObjC)"))
+  .code {
+#if _runtime(_ObjC)
```

Comment:
The rest of this file chooses this route.  Should I refactor the rest of the tests?